sdk/translation/azure-ai-translation-text/samples/README.md

```diff
@@ -26,30 +26,68 @@ See the [README][README] of the Text Translator client library for more informat
 
 ## Common scenarios samples
 
-
 # Create Client
 
-For some of these operations you can create a new `TextTranslationClient` without any authentication. You will only need your endpoint:
+You can create Translator resource following [Create a Translator resource][translator_resource_create].  For general information about authentication with a cognitive services resource please refer to the documentation [here][cognitive_auth].
```

Comment:
Same comment for "[here][cognitive_auth]."

Reply:
Removed.